### PR TITLE
docs: fix language "if challenge is not known"

### PR DESCRIPTION
**Pull request by amosshapira**
### More

- [ ] Added/updated tests
- [x] Added/updated documentation

### CODE REVIEW

File: docs/user-guide/cluster-docker-consul.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ If you want to use Let's Encrypt with Traefik, sharing configuration or TLS cert
 Ok, could we mount a shared volume used by all my instances? Yes, you can, but it will not work.
 When you use Let's Encrypt, you need to store certificates, but not only.
 When Traefik generates a new certificate, it configures a challenge and once Let's Encrypt will verify the ownership of the domain, it will ping back the challenge.
-If the challenge is not known by other Traefik instances, the validation will fail.
+If the challenge is not recognised by other Traefik instances, the validation will fail.
```
Comment (dtomcej) [inline docs/user-guide/cluster-docker-consul.md:12]:
```suggestion
If the challenge is not recognized by other Traefik instances, the validation will fail.
```

**Review by kevinpollet — APPROVED** [review]
Thanks 👍

**Review by jbdoumenjou — APPROVED** [review]
LGTM :+1:

**Review by jspdown — APPROVED** [review]
LGTM :+1:

**Pull request merged**